examples/components/pubsub/kafka.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: kafka-pubsub
+spec:
+  type: pubsub.kafka
+  version: v1
+  metadata:
+    - name: brokers # Required. Kafka broker connection setting
+      value: "localhost:9092"
+    - name: "authType"
+      value: "none"
```

Comment:
One thing I would like to improve in these ITs, is to scope components only to applicable apps. Can you add scope here, so this component is not used by other tests?

Reply:
done

Reply:
Oh. I mean to use the actual scope feature - not move to a different folder.